src/zjs_sensor.c

```diff
@@ -450,7 +496,7 @@ static jerry_value_t zjs_sensor_create(const jerry_value_t function_obj,
                                        enum sensor_channel channel)
 {
     double frequency = DEFAULT_SAMPLING_FREQUENCY;
-    uint32_t pin;
+    uint32_t pin = -1;
```

Comment:
Let's not assign negative numbers to unsigned ints, even if it works. :)

I guess use 0xffffffff?